components/WithdrawLiquidity/WithdrawLiquidityController.tsx

```diff
@@ -61,7 +62,12 @@ const WithdrawLiquidity = () => {
   }
 
   async function withdrawLiquidity(balance: number) {
-    let amt = ethers.utils.parseEther(((liquidityAmt * balance) / 100).toString()).toString()
+    let amt = '0'
+    if (balance === 100) {
+      amt = totalLiquidityAmt.toString()
+    } else {
+      amt = ethers.utils.parseEther(((liquidityAmt * balance) / 100).toString()).toString()
+    }
     if (provider) {
       let address = '0x3eFadc5E507bbdA54dDb4C290cc3058DA8163152'
```

Comment:
Can you add a comment here, what this address is?

Reply:
Done. (It is deposit poll contract address)

Reply:
Ok, it's also for ropsten right?

Reply:
Yes I got this address from the abi of Deposit pool contract abi

Reply:
Should mention it's from ropsten.